packages/frontend/src/page/menubar.tsx

```diff
@@ -4,14 +4,13 @@ import { getAuth, signOut } from "firebase/auth";
 import { useAuth, useFirebaseApp } from "solid-firebase";
 import { type JSX, Show, createSignal } from "solid-js";
 
-import { Dialog, IconButton } from "../components";
-import { Login } from "../user";
-
-import CircleHelp from "lucide-solid/icons/circle-help";
+import Info from "lucide-solid/icons/info";
 import LogInIcon from "lucide-solid/icons/log-in";
 import LogOutIcon from "lucide-solid/icons/log-out";
 import MenuIcon from "lucide-solid/icons/menu";
 import SettingsIcon from "lucide-solid/icons/settings";
+import { Dialog, IconButton } from "../components";
+import { Login } from "../user";
```

Comment:
Can you put these last two imports back where they were? I like to keep the icon imports in their own "section" but unfortunately the editor tools that automatically insert imports are too dumb for put things in exactly the right place.